transform/mattermost-analytics/models/marts/product/trial_requests/fct_trial_request_history.sql

```diff
@@ -40,6 +40,9 @@ select
     , tr.end_at
     , tr.request_source
     , tr.request_type
+    , tr.product_id
```

Comment:
For now, perhaps we can rename this to `stripe_product_id` to be more explicit about the origin of this product.